src/internal/user_api.rs

```diff
@@ -257,9 +258,78 @@ impl UserDevice {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+struct Claims {
+    sub: String,
+    pid: usize,
+    sid: String,
+    kid: Option<usize>,
```

Comment:
While this is optional on the ironcore-id side, I'd vote that we don't let people omit this as it's in their best interest to not to it. The `kid` being optional has probably never really been shown as an option in IronOxide as we changed it over to be required before this repo existed.

Reply:
I agree. I don't think there's any value in letting someone omit `kid`. It would be nice to eventually omit the code that allows ahat on the server side at some point.

Reply:
Sounds good to me. Done